app/src/main/java/com/xmartlabs/scasas/criminalintent/ui/crime/list/CrimeListFragment.java

```diff
@@ -83,8 +90,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
             .solved(false)
             .title("")
             .build();
-        CrimeController.getInstance().insertCrime(crime);
-        startActivity(getCrimePagerActivityIntent(crime));
+        List<Crime> crimes = new ArrayList<>();
+        crimes.add(crime);
+        startActivity(getCrimePagerActivityIntent(crime, crimes, true));
```

Comment:
you shouldn't send a crime. The CrimePagerActivity should realize that the you are crating a new crime, if you don't pass any parameter. The CrimePagerActivity should create the crime